Source/Core/UICommon/UICommon.cpp

```diff
@@ -551,10 +553,8 @@ std::string FormatSize(u64 bytes, int decimals)
 
   // Don't need exact values, only 5 most significant digits
   const double unit_size = std::pow(2, unit * 10);
-  std::ostringstream ss;
-  ss << std::fixed << std::setprecision(decimals);
-  ss << bytes / unit_size << ' ' << Common::GetStringT(unit_symbols[unit]);
-  return ss.str();
+  return fmt::format("{:.{}f} {}", bytes / unit_size, decimals,
+                     Common::GetStringT(unit_symbols[unit]));
```

Comment:
This needs to be `{:.{}Lf} {}` to be equivalent since this previously respected the user's locale for the decimal separator.